src/app/avatar/heygen/index.tsx

```diff
@@ -0,0 +1,174 @@
+'use client'
+import React, { useState, useEffect, useRef, useCallback } from 'react';
+import { handleICE, newSession, repeat, startSession, stopSession } from './helper';
+
+// Interface for session info
+interface SessionInfo {
+  session_id: string;
```

Comment:
I think this is really the only data you need to store, not the other two fields.